hudi-hadoop-common/src/main/java/org/apache/hudi/storage/hadoop/HadoopStorageConfiguration.java

```diff
@@ -66,18 +67,6 @@ public Configuration newCopy() {
     return new Configuration(configuration);
   }
 
-  @Override
-  public void writeObject(ObjectOutputStream out) throws IOException {
-    out.defaultWriteObject();
-    configuration.write(out);
-  }
```

Comment:
We already have a `SerializableConfiguration`, does these two have similiar functionalities?

Reply:
We’ll remove `SerializableConfiguration`.

Reply:
i.e., `HadoopStorageConfiguration` replaces `SerializableConfiguration ` eventually.